fennel/client_tests/test_dataset.py

```diff
@@ -3177,3 +3178,64 @@ def test_lookup_as_of_time(client):
     assert len(data) == 2
     assert data[0]["name"] is None
     assert data[1]["name"] == "Monica"
+
+
+@pytest.mark.integration
+@mock
+def test_erase_key(client: Client):
+    client.sync(datasets=[UserInfoDataset, UserInfoDatasetDerived])
+    data = [
+        [18232, "Ross", 24, "USA", "2022-11-10 01:22:23"],
+        [18233, "Monica", 24, "USA", "2022-11-15 01:33:13"],
+        [18234, "Chandler", 24, "USA", "2022-11-16 01:22:23"],
+        [18235, "Rachel", 24, "USA", "2022-11-17 01:33:13"],
+    ]
+    columns = ["user_id", "name", "age", "country", "timestamp"]
+    df = pd.DataFrame(data, columns=columns)
+
+    response = client.log("fennel_webhook", "UserInfoDataset", df)
+    assert response.status_code == requests.codes.OK, response.json()
+
+    data, found = client.lookup(
+        "UserInfoDatasetDerived",
+        [{"user_id": 18232}],
+        fields=["name"],
+    )
+    assert len(found.tolist()) == 1
+    assert found.tolist() == [True]
+    assert len(data) == 1
+    assert data[0]["name"] == "Ross"
+
+    # Should be deleted
+
+    response = client.erase_key("UserInfoDatasetDerived", [{"user_id": 18232}])
```

Comment:
This method should simply be called `erase` instead of `erase_key`

Reply:
Done